src/coreclr/src/zap/zapcode.h

```diff
@@ -216,7 +216,7 @@ class ZapCodeMethodDescs : public ZapNode
     {
     }
 
-    virtual UINT GetAligment()
+    virtual UINT GetAlignment()
```

Comment:
It was not overriding `ZapNode::GetAlignment` due to this typo.

Reply:
@davidwrighton  .. not sure who owns this file.

Reply:
I'll say I own it. And this is a good change.

Reply:
I guess this wouldn't happen in C#, thanks to the `override` keyword

Reply:
We have C++11 [`override`](https://en.cppreference.com/w/cpp/language/override) specifier as well for which compilers provide good analysis. I will try to follow up on it (in a separate PR).